run_portal.ps1

```diff
@@ -0,0 +1,27 @@
+# Change to uid2-admin directory, run docker compose and then maven compile
+Set-Location -Path "../uid2-admin" -ErrorAction Stop
+docker compose up -d
```

Comment:
`-d` is for detached mode, which starts the containers in the background and leaves them running. This frees up the terminal so it can keep going.

Source: https://docs.docker.com/reference/cli/docker/compose/up/